CRITICAL - 2021-02-13 15:08:23 --> Unable to connect to the database.
#0 C:\laragon\www\cms\vendor\codeigniter4\framework\system\Database\BaseConnection.php(618): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\laragon\www\cms\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1866): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#2 C:\laragon\www\cms\vendor\codeigniter4\framework\system\Model.php(169): CodeIgniter\Database\BaseBuilder->get()
#3 C:\laragon\www\cms\vendor\codeigniter4\framework\system\BaseModel.php(518): CodeIgniter\Model->doFind(true, 1)
#4 C:\laragon\www\cms\app\Helpers\configuration_helper.php(8): CodeIgniter\BaseModel->find(1)
#5 C:\laragon\www\cms\app\Views\auth\login.php(10): configInfo()
#6 C:\laragon\www\cms\vendor\codeigniter4\framework\system\View\View.php(220): include('C:\\laragon\\www\\...')
#7 C:\laragon\www\cms\vendor\codeigniter4\framework\system\View\View.php(222): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#8 C:\laragon\www\cms\vendor\codeigniter4\framework\system\Common.php(1217): CodeIgniter\View\View->render('auth/login', Array, true)
#9 C:\laragon\www\cms\app\Controllers\AuthController.php(15): view('auth/login')
#10 C:\laragon\www\cms\vendor\codeigniter4\framework\system\CodeIgniter.php(936): App\Controllers\AuthController->login()
#11 C:\laragon\www\cms\vendor\codeigniter4\framework\system\CodeIgniter.php(432): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
#12 C:\laragon\www\cms\vendor\codeigniter4\framework\system\CodeIgniter.php(333): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#13 C:\laragon\www\cms\public\index.php(44): CodeIgniter\CodeIgniter->run()
#14 C:\laragon\www\cms\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(45): require_once('C:\\laragon\\www\\...')
#15 {main}
